src/components/modal/index.tsx

```diff
@@ -10,15 +10,17 @@ export type ModalStyle = "white" | "dark"
 
 interface Props {
   close: () => void
+  onClose?: () => void
```

Comment:
couln't you add code to the close function instead?

```
<Modal close={() => close(); afterClose(); }
```

Reply:
ah, that would not cover the case where we rely on the "internal close"

Reply:
Wouldn't that break the interface of the component and we would need to update it everywhere over two projects?